src/components/themes/themes.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '@mapbox/mr-ui/icon';
+
+class Image extends React.Component {
+  render() {
+    const { color, icon } = this.props;
+    return (
+      <div
+        className={`bg-${color}-light round-full color-${color} flex-parent flex-parent--center-main flex-parent--center-cross`}
+        style={{ width: 50, height: 50 }}
+      >
+        <Icon name={icon} size="40" />
+      </div>
+    );
+  }
+}
+
+Image.propTypes = {
+  color: PropTypes.oneOf([
+    'gray',
+    'orange',
+    'red',
+    'green',
+    'purple',
+    'pink',
+    'yellow',
+    'blue'
+  ]).isRequired,
+  icon: PropTypes.string
+};
+
+const themes = {
+  default: {
+    image: <Image icon="book" color="gray" />,
+    label: 'Note',
+    styles: {
+      background: '#f4f7fb',
+      color: '#547190'
+    }
+  },
+  warning: {
+    image: <Image icon="alert" color="orange" />,
+    label: 'Warning',
+    styles: {
+      background: '#feefe2',
+      color: '#945823',
+      borderColor: '#dea573'
+    }
+  },
+  error: {
+    image: <Image icon="alert" color="red" />,
+    label: 'Error',
+    styles: {
+      background: '#fbe5e5',
+      color: '#ba3b3f'
+    }
+  },
+  beta: {
+    image: <Image icon="marker" color="green" />,
+    label: 'Beta',
+    tooltipText: 'This feature is in public beta and is subject to changes.',
+    styles: {
+      background: '#e8f5ee',
+      color: '#1b7d4f',
+      borderColor: '#75c684'
+    }
+  },
+  download: {
+    image: <Image icon="arrow-down" color="purple" />,
+    label: 'Download',
+    styles: {
+      background: '#f2effa',
+      color: '#5a3fc0'
+    }
+  },
+  fundamentals: {
+    label: 'Fundamentals',
+    tooltipText:
+      'The concepts described here are fundamental to using this product.',
+    styles: {
+      background: '#fff0f7',
+      color: '#cf1c61',
+      borderColor: '#fd8ac0'
+    }
+  }
+};
+/* duplicate themes */
+themes.legacy = Object.assign({}, themes.warning);
+themes.legacy.label = 'Legacy';
+(themes.legacy.tooltipText =
+  'This feature is no longer in active development.'),
+  (themes.new = Object.assign({}, themes.beta));
```

Comment:
Now that the default theme is gray (and blue is available), what do you think about using blue for either `new` or `beta`?

Reply:
wowowowowow blue for beta!!!!!!!

Reply:
I like the blue! Just want to flag that the color of links is the same color as the text:

![image](https://user-images.githubusercontent.com/2180540/71701349-e8a9ae00-2d96-11ea-9476-a56abb1d93e1.png)

@colleenmcginnis I think it's fine since links are underscored, but wanted to double check with you!

Reply:
Ohhh and icon, maybe plus symbol for new?

![image](https://user-images.githubusercontent.com/2180540/71701509-c9f7e700-2d97-11ea-9fcd-3377e2cb88d4.png)

Reply:
>@colleenmcginnis I think it's fine since links are underscored, but wanted to double check with you!

I think /help pages have custom CSS that removed the underline 😬 but maybe we should change that for consistency anyway!

Reply:
@colleenmcginnis yeah let's make sure /help underscores links since that's a best practice 🤠